api/core/listen_result.go

```diff
@@ -37,39 +37,38 @@ func (s *Server) ListenResult(request *ListenResultRequest, stream Core_ListenRe
 			})
 		}
 	}
-	return
+	return nil
 }
 
-func validateTaskKey(service *service.Service, taskKey string) (err error) {
+func validateTaskKey(service *service.Service, taskKey string) error {
 	if taskKey == "" || taskKey == "*" {
-		return
+		return nil
 	}
 	_, ok := service.Tasks[taskKey]
 	if ok {
-		return
+		return nil
 	}
-	err = errors.New("Task '" + taskKey + "' doesn't exist in this service")
-	return
+	return fmt.Errorf("Task '%q' doesn't exist in this service", taskKey)
```

Comment:
remove ' around %q. %q already puts double quotes around the world.

Reply:
Done